pkg/bindinfo/BUILD.bazel

```diff
@@ -58,7 +58,7 @@ go_test(
     embed = [":bindinfo"],
     flaky = True,
     race = "on",
-    shard_count = 41,
+    shard_count = 43,
```

Comment:
Where is 42 ?

Reply:
Because I added 2 new test cases in `pkg/bindinfo/global_handle_test.go`, it becomes from 41 to 43.